lib/plugins/aws/utils/resolveCfFunctionValue.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const memoizee = require('memoizee');
+const resolveCfImportValue = require('./resolveCfImportValue');
+
+const CF_IMPORT_VALUE_KEY = 'Fn::ImportValue';
+
+function resolveCfFunctionValue(provider, cfFunction) {
```

Comment:
Are you sure such generalization is useful at this point?

I think I wouldn't do it until we come up with a valid plan on how we can  support other CF functions (at this point it's not clear how well they can be supported, and if at all adding such support makes sense).

I would simply support CF import resolver if we detect that env var is CF import (do not fall into generalization until we need it)